app/src/main/java/com/locoquest/app/dto/User.kt

```diff
@@ -2,4 +2,5 @@ package com.locoquest.app.dto
 
 data class User(
     val uid: String,
-    val displayName: String)
+    val displayName: String
+)
```

Comment:
Reformatted some code in the DTO classes to clean up the general look of the code, according to how its presented in the documentation.
https://kotlinlang.org/docs/coding-conventions.html#class-headers

Reply:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.